validate_data.py

```diff
@@ -117,8 +120,7 @@ def nullify_and_prune(data, endpoint):
 def postprocess(data, endpoint, titles=None):
     """Nullify and prune data, add titles, and return 404 if appropriate"""
     pruned_data = nullify_and_prune(data, endpoint)
-
-    if pruned_data in [{}, None, 0]:
+    if pruned_data in [{}, None, 0] or None in pruned_data:
```

Comment:
Same thought as prior comment, I can't quite see what cases this covers!

Reply:
Same thing as the previous one! This was for a now defunct use case.